gradle/owasp-suppression.xml

```diff
@@ -13,6 +13,13 @@
         <packageUrl regex="true">^pkg:maven/io\.netty/netty*@*.*$</packageUrl>
         <vulnerabilityName>CVE-2023-4586</vulnerabilityName>
      </suppress>
+    <suppress>
+        <notes><![CDATA[
+        Suppress CVE-2023-36415 as this should only be applicable on version up to but excluding 1.10.2.
```

Comment:
Is that a mistake in the metadata then? Might be worth adding `until` if we could eventually remove this suppression, just so we can keep the file clean